packages/seatmaps-client/src/index.tsx

```diff
@@ -0,0 +1,80 @@
+import "unfetch/polyfill";
```

Comment:
I think it's better to use single quotes instead of double ones throughout the project like we do it in the Shell project.  Anyway it's a minor comment

Reply:
I think better use same prettier configs in all projects

Reply:
I've tried to limit style changes compared to the previous version, but I agree with you guys on the consistency argument.
Will change the prettier config in a separate PR.